src/usdb_syncer/gui/forms/MainWindow.ui

```diff
@@ -594,6 +727,31 @@
     <string>Show Log</string>
    </property>
   </action>
+  <action name="action_songs_download">
+   <property name="text">
+    <string>Download</string>
+   </property>
+  </action>
+  <action name="action_songs_to_batch">
+   <property name="text">
+    <string>Add to Batch</string>
```

Comment:
Add to [download] queue?

Reply:
Yeah, I was a bit inconsistent there. Maybe we should use _Batch_ everywhere? I feel like _Download Queue_ could also refer to the songs that are currently being downloaded (`DownloadStatus.DOWNLOADING`) or waiting for a thread (`DownloadStatus.PENDING`) only.